invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActions.js

```diff
@@ -0,0 +1,34 @@
+import { RequestLinkExtractor } from "../../api";
+import React from "react";
+import Overridable from "react-overridable";
+import ReactDOM from "react-dom";
+import { RequestAction } from "./index";
+
+const element = document.getElementById("request-actions");
+
+export const RequestActions = ({ request }) => {
+  const actions = Object.keys(new RequestLinkExtractor(request).actions);
+  return (
+    <Overridable id="InvenioRequests.RequestActions.layout" request={request}>
+      <>
+        {actions.map((action, url) => (
```

Comment:
URL unused? Isn't it the map given only one element with `Object.keys`?
```suggestion
        {actions.map((action) => (
```